cloudapi/logs.go

```diff
@@ -100,6 +100,8 @@ func (c *Config) logtailConn(ctx context.Context, referenceID string, since time
 
 	headers := make(http.Header)
 	headers.Add("Sec-WebSocket-Protocol", "token="+c.Token.String)
+	headers.Add("Authorization", "token "+c.Token.String)
+	headers.Add("X-K6TestRun-Id", referenceID)
```

Comment:
Does it mean that if currently I run two test I will get *both* of their logs?

Reply:
Nope! The `test_run_id` label is already making sure that doesn't happen. Without going to much into details, the new headers added are part of the more generic auth layer. In the end though, all information received is used to authenticate+authorize.